README.md

```diff
@@ -1,101 +1,39 @@
-# HTML & CSS3 & JavaScript Course
+# microverse-capstone-project-one
 
-If you are not familiar with linters, read [root level README](../README.md).
+> This is my capstone project on html + css+ javascript. It's Topic is the Global Conference on Renewable Energy and Climate Change. The speakers section is loaded dynamically using javascript.
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- html 
+- CSS
+- Javascript
+- Bootstrap
+- Linter
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+## Link to project online
+- [OnlineVersion](https://ceemos96.github.io/MV-capstone-project/)
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+## Link to project recorded video
+- [RecordedVideo](#)
```

Comment:
- [ ]  I see you have the video link in the PR description, great! However, it would be better to update this link too, as the previous reviewers said.